app/src/main/res/layout/fragment_urlaub_edit.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <EditText
+        android:id="@+id/urlaub_edit_edit_datum_bis"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:autofillHints="@string/haushaltedithint_datum"
+        android:ems="10"
+        android:hint="@string/haushaltedithint_datum"
+        android:inputType="date"
+        android:textSize="@dimen/content_text_big"
+        app:layout_constraintStart_toStartOf="@+id/urlaub_edit_text_datum_bis"
+        app:layout_constraintTop_toBottomOf="@+id/urlaub_edit_text_datum_bis" />
+
+    <Button
+        android:id="@+id/urlaub_edit_button_abbrechen"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginBottom="16dp"
+        android:background="@color/colorDarkGreen"
+        android:text="@string/button_abbrechen"
+        android:textColor="@color/colorWhite"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <Button
+        android:id="@+id/urlaub_edit_button_delete"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="16dp"
```

Comment:
Hier hättest du noch einiges in dimens auslagern können. Und da du die API auf 19 gesetzt hast, kannst du das margin Left und Right weg lassen

Reply:
Wird beim nächsten commit gefixt.